modules/dr-recover-expired-control-plane-certs.adoc

```diff
@@ -5,167 +5,33 @@
 [id="dr-scenario-3-recovering-expired-certs_{context}"]
 = Recovering from expired control plane certificates
 
-Follow this procedure to recover from a situation where your control plane certificates have expired.
+As of {product-title} 4.5, the cluster can automatically recover from expired control plane certificates. You no longer need to perform the manual steps that were required in previous versions.
 
-.Prerequisites
+The exception is that you must manually approve the pending `node-bootstrapper` certificate signing requests (CSRs) to recover kubelet certificates.
 
-* SSH access to master hosts.
+Use the following steps to approve the pending `node-bootstrapper` CSRs:
```

Comment:
Should this come after the `.Procedure`? If not, s/CSRs:/CSRs.?

Reply:
Do the `node-bootstrapper`CSRs have `node-bootstrapper` in the name?

Reply:
Yes they do